tbx/static_src/sass/components/_logo.scss

```diff
@@ -37,7 +37,7 @@
 
         @include media-query(large) {
             width: 227px;
-            height: 41px;
+            height: 40px;
```

Comment:
I'm still seeing a small jump in the position of the public logo. I think it is down to how it is cropped, so we might need to ask a designer to amend the svg.

Reply:
Thanks! I've updated the svg's viewbox so that the cropping is closer to the other logos. It should look fine now.